lib/src/js_utils.dart

```diff
@@ -49,7 +49,7 @@ class JsEventListener {
 
   JsEventListener(this._proxy, this._name, {this.cvtEvent, this.twoArgs: false});
 
-  Stream get stream {
+  Stream<T> get stream {
```

Comment:
You could probably parameterize the `StreamController _controller` field as well.

Reply:
done